portfolio/src/main/webapp/script2.js

```diff
@@ -0,0 +1,12 @@
+/*
+*This talks about me!
+*/
+
+function addBackground()  {
```

Comment:
Any reason for having this in a separate file? I noticed you had something similar in the other script file - was it not working over there?

Reply:
Yeah it was not working over there. Although I found out it was because I was just bringing the writing to a new line so I could see it all and js did not like that.